files/templates/mbta/email/messages/messages_pt.properties

```diff
@@ -1,6 +1,6 @@
 secondFactor.emailSubject=Código de verificação MBTA
 secondFactor.emailbody=Seu código de verificação é: {0}
```

Comment:
question: should this be `emailBody` (capital-B)?

Reply:
yes, fixed in https://github.com/mbta/keycloak-deploy/pull/37/commits/9f6315ec968af070bffadddb5650032f7af38f55